judoscale-ruby/lib/judoscale/config.rb

```diff
@@ -99,11 +85,10 @@ def reset
 
       if ENV["RENDER_INSTANCE_ID"]
         instance = ENV["RENDER_INSTANCE_ID"].delete_prefix(ENV["RENDER_SERVICE_ID"]).delete_prefix("-")
-        @current_runtime_container = RuntimeContainer.new ENV["RENDER_SERVICE_ID"], instance
+        @current_runtime_container = RuntimeContainer.new instance
         @api_base_url ||= "https://adapter.judoscale.com/api/#{ENV["RENDER_SERVICE_ID"]}"
       elsif ENV["DYNO"]
-        service_name, instance = ENV["DYNO"].split "."
-        @current_runtime_container = RuntimeContainer.new service_name, instance
+        @current_runtime_container = RuntimeContainer.new ENV["DYNO"]
       else
         # unsupported platform? Don't want to leave @current_runtime_container nil though
```

Comment:
Might want to drop this comment now 🤔

Reply:
Is it no longer valid? I still found it helpful, assuming we don't want `@current_runtime_container` to be nil.

Reply:
We don't, but the "unsupported platform" bit isn't a valid assumption anymore, now that we know there are platforms which the _correct_ functionality is to have the bare `RuntimeContainer.new` 🤔 so maybe just the first 2 words of the comment 😆